ansible/roles/test/tasks/bgp_neighbor/bgp_neighbor_test_ipv4_bounce_neighbor.yml

```diff
@@ -0,0 +1,49 @@
+# Subtask to bounce an ipv4 neighbor and verify it comes back
+- block:
+  - name: output variables
+    debug:
+      msg: "Exercising BGP neighbor link: {{ neighbor }}"
+
+  - name: shut down neighbor link {{ neighbor }} 
+    shell: config bgp shutdown neighbor {{ neighbor }}
```

Comment:
So, we aren't really shutting down the interface. Instead we are just resetting the peer?

Reply:
This was the test that we were requested to write.